esileapclient/tests/functional/v1/test_basic.py

```diff
@@ -213,6 +214,36 @@ def test_lease_show_offer_deleted(self):
         details = esi.lease_show(self.clients['child-lessee'], lease['uuid'])
         self.assertEqual(details['status'], 'deleted')
 
+    def test_node_list_long(self):
+        """ Tests functionality of "esi node list" if using node_uuid.
```

Comment:
That was my argument, Then why do we need different test just for node name. Why can't we just test with `openstack esi node list --long`.

Reply:
My line of thinking is that since we want the functional tests to simulate requests real users would make, and since `openstack esi node list` without the `--long` flag invokes the "default" behavior of the command, we should therefore include at least one test that runs without `--long` to validate said default behavior.

Reply:
Updated in the latest push